src/app/core/app.component.html

```diff
@@ -37,8 +34,9 @@
       </div>
       <alg-top-right-controls [drawLeftBorder]="currentMode === 'normal'"></alg-top-right-controls>
     </div>
-    <div class="right" [ngClass]="{ collapsed: !leftMenuDisplayed, expanded: leftMenuDisplayed, 'with-mode-bar': currentMode !== 'normal' }">
-      <div class="main-content" [ngClass]="{ folded: !headersDisplayed, unfolded: headersDisplayed }">
+    <div class="right"
+      [ngClass]="{ collapsed: !leftMenuAndHeaders, expanded: leftMenuAndHeaders, 'with-mode-bar': currentMode !== 'normal' }">
+      <div class="main-content" [ngClass]="{ folded: !leftMenuAndHeaders, unfolded: leftMenuAndHeaders }">
```

Comment:
I suppose it should inherit from the parent div with expanded/collapsed instead of defining 2 conditions again here.

Reply:
Done, thanks.